packages/benchmark.js-plugin/src/index.ts

```diff
@@ -93,7 +92,11 @@ function withCodSpeedBenchmark(bench: Benchmark): WithCodSpeedBenchmark {
     };
     return bench;
   }
-  const callingFile = getCallingFile();
+  const callingFile = getCallingFile(3);
```

Comment:
Quite bad magic number here,  a comment with the stack structure (similar to the old one) will help